source/agora/common/TransactionPool.d

```diff
@@ -221,7 +225,15 @@ unittest
     txs.each!(tx => pool.add(tx));
     assert(pool.length == txs.length);
 
-    auto pool_txs = pool.take(txs.length);
```

Comment:
Rule of three: provide a `version(unittest)` wrapper ?

Reply:
Well now that we have code coverage, I'm on board with that idea.